debian/gbp.conf

```diff
@@ -0,0 +1,14 @@
+# Configuration file for git-buildpackage and friends
+
+[DEFAULT]
+compression = gz
+debian-branch = debian/sid
```

Comment:
We don't want to have our `.deb` file mention a specific release of Debian.  This file can probably just be dropped.  Debian can do their own release-specific thing.